middleware/authorization.js

```diff
@@ -6,18 +6,19 @@ const bcrypt = require("bcrypt");
 
 async function issueToken (req, res) {
     const { body } = req;
-    const row = await userRepository.findUser_email(body.email);
-    const tutee = await tuteeRepository.findTuteeId(row.id);
+    const row = await userRepository.findUserEmail(body.email);
 
     if (!row) {
-        return res.status(400).json( {message: "해당 이메일이 없습니다."});
+        return res.status(400).json({ message: "해당 이메일이 없습니다." });
     }
 
+    const tutee = await tuteeRepository.findTuteeId(row.id);
+
     if (!tutee.google_id) {
         const isPw = bcrypt.compareSync(body.pw, row.pw);
 
         if (!isPw) {
-            return res.status(400).json({ message: "잘못된 비밀번호 입니다."});
+            return res.status(400).json({ message: "잘못된 비밀번호 입니다." });
```

Comment:
```suggestion
            return res.status(401).json({ message: "잘못된 비밀번호 입니다." });
```
400은 잘못된 요청을 보냈을때 사용합니다. 

**400 Bad Request**
> `Content-Type` 형식을 지원하지 않을때
> `Accept` 형식을 지원하지 않을때
> 필수 필드가 없을때
> 악의적인 요청이 감지 되었을때 등등...

401 Unauthorized
> 인증되지 않은 클라이언트 접근